addons/gunbag/data/model.cfg

```diff
@@ -7,119 +7,118 @@ class CfgSkeletons {
+                                                    "Face_chopRight","Face_Jawbone",
+                                                    "Face_chopLeft","Face_Jawbone",
+                                                    "Face_LipLowerMiddle","Face_Jawbone",
+                                                    "Face_LipLowerLeft","Face_Jawbone",
+                                                    "Face_LipLowerRight","Face_Jawbone",
+                                                    "Face_Chin","Face_Jawbone",
+                                                    "Face_Tongue","Face_Jawbone",
+                                                "Face_CornerRight","Face_Hub",
+                                                    "Face_CheekSideRight","Face_CornerRight",
+                                                "Face_CornerLeft","Face_Hub",
+                                                    "Face_CheekSideLeft","Face_CornerLeft",
+                                                "Face_CheekFrontRight","Face_Hub",
+                                                "Face_CheekFrontLeft","Face_Hub",
+                                                "Face_CheekUpperRight","Face_Hub",
+                                                "Face_CheekUpperLeft","Face_Hub",
+                                                "Face_LipUpperMiddle","Face_Hub",
+                                                "Face_LipUpperRight","Face_Hub",
+                                                "Face_LipUpperLeft","Face_Hub",
+                                                "Face_NostrilRight","Face_Hub",
+                                                "Face_NostrilLeft","Face_Hub",
+                                                "Face_Forehead","Face_Hub",
+                                                    "Face_BrowFrontRight","Face_Forehead",
+                                                    "Face_BrowFrontLeft","Face_Forehead",
+                                                    "Face_BrowMiddle","Face_Forehead",
+                                                    "Face_BrowSideRight","Face_Forehead",
+                                                    "Face_BrowSideLeft","Face_Forehead",
+                                                "Face_Eyelids","Face_Hub",
+                                                "Face_EyelidUpperRight","Face_Hub",
+                                                "Face_EyelidUpperLeft","Face_Hub",
+                                                "Face_EyelidLowerRight","Face_Hub",
+                                                "Face_EyelidLowerLeft","Face_Hub",
+                                                "EyeLeft","Face_Hub",
+                                                "EyeRight","Face_Hub"
+        };// end of skeleton array
         // location of pivot points (local axes) for hierarchical animation
-        pivotsModel="A3\anims_f\data\skeleton\SkeletonPivots.p3d";
```

Comment:
Should stay the `A3\anims_f` path, for ever instance of the OFP2_ManSkeleton that we have